src/lib/i18n/numbers.ts

```diff
@@ -76,19 +76,23 @@ export function toLocalFixed(value: BigNumber.Value, decimalPlaces?: number, rou
 
 export function toShortened(value: BigNumber.Value) {
   let bn = new BigNumber(value);
-  if (bn.abs().lt(1)) {
-    return toLocalFixed(bn.toPrecision(1));
-  }
+  const target = bn.abs().decimalPlaces(2);
+
+  if (target.lt(0.01)) return toLocalFixed(bn.toPrecision(2));
+
+  if (target.lt(10_000)) return toLocalFixed(bn, 2);
```

Comment:
I heard that style of numbers like that `10_000` could lead to bugs in some cases

Reply:
So there are reasons for that..